examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.java

```diff
@@ -111,7 +112,12 @@ public void setFailure(String error) {
   }
 
   public void waitCompleteMs(long timeoutMs) {
+    logger.log(Level.INFO, "waitCompleteMs is called ");
     mFutureResult.setTimeoutMs(timeoutMs);
     mFutureResult.waitResult();
   }
+
+  public void clear() {
```

Comment:
Side note: It would be good to make the methods in this class final, since the class is abstract (so subclasses can't accidentally change the behavior via overrides).

Reply:
updated, thanks